conda-forge.yml

```diff
@@ -2,3 +2,7 @@ github:
   branch_name: main
   tooling_branch_name: main
 conda_forge_output_validation: true
+
+build_platform:
+  osx_arm64: osx_64
+test: native_and_emulated
```

Comment:
Are you sure that's really what you want?  Is it going to be able to run tests when cross compiling?

Reply:
I was following this blog post: https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/

I've never tried to manually get the osx_arm builds to fire off manually, I always use the bot method.

What makes this annoying is the only reason we can't keep this package `noarch` (which would make osx_arm support work out of the box) is the selector we need to make NNPops not a dependency for osx...

Maybe we just go back to `noarch` and remove the nnpops dependency entirely? We can add a note in the README that if you are on linux, you should install nnpops. 

Thoughts @peastman ?

Reply:
> I've never tried to manually get the osx_arm builds to fire off manually, I always use the bot method.

I was just referring to the `test` option.  Should it be set to `native`?

> Maybe we just go back to noarch and remove the nnpops dependency entirely?

In the short term we can probably work with that.  In the long term, we'll need some way to make sure it gets installed on Linux, or people will end up getting much slower performance.